.github/workflows/build-docker-artifact.yaml

```diff
@@ -1,18 +1,26 @@
 name: "Build tt-metal docker artifact"
 
 on:
-  workflow_call:
   workflow_dispatch:
-
+  push:
+    branches:
+      - main
+    paths:
+      - dockerfile/ubuntu-20.04-amd64.Dockerfile
+      - scripts/docker/requirements*
+      - pyproject.toml
+      - scripts/docker/install_test_deps.sh
+      - tt_metal/python_env/requirements-dev.txt
```

Comment:
Mind opening an issue to add requirements-infra to this list and assigning to me? Don't wanna block

Reply:
https://github.com/tenstorrent/tt-metal/issues/12233